### PR TITLE
Enable test_det_logdet_slogdet on ROCm

### PR DESCRIPTION
**Pull request by BLOrange-AMD**

### CODE REVIEW

File: test/test_linalg.py [hunk 1]
```diff
@@ -6387,7 +6387,6 @@ def test_slogdet_errors_and_warnings(self, device, dtype):
     # FIXME One of the backends of lu_factor fails in windows. I haven't investigated which or why
     # https://github.com/pytorch/pytorch/issues/75225
     @unittest.skipIf(IS_WINDOWS, "Skipped on Windows!")
-    @skipCUDAIfNoCusolver
```
Comment (pruthvistony) [inline test/test_linalg.py:6390]: This decorator should be `@skipCUDAIfNoHipsolverOrCusolver`

**Review by pruthvistony — CHANGES_REQUESTED** [review]

**Comment by pruthvistony** [comment]
jenkins please retest

**Comment by pruthvistony** [comment]
jenkins retest this please

**Comment by BLOrange-AMD** [comment]
jenkins retest this please

**Comment by pruthvistony** [comment]
jenkins retest this please

**Comment by BLOrange-AMD** [comment]
jenkins retest this please

**Pull request closed without merging**